moto/elbv2/models.py

```diff
@@ -833,6 +837,11 @@ def create_target_group(self, name, **kwargs):
         self.target_groups[target_group.arn] = target_group
         return target_group
 
+    def convert_and_validate_certificates(self, certificate):
+
+        # transform default certificate  to conform with the rest of the code and XML templates
+        default_cert = {"certificate_arn": certificate[0]["CertificateArn"]}
```

Comment:
Missing `return` statement.
Also, both [`convert_and_validate_action_properties`](https://github.com/spulec/moto/blob/master/moto/elbv2/models.py#L612) and [`convert_and_validate_condition_propertie`](https://github.com/spulec/moto/blob/master/moto/elbv2/models.py#L661) return lists. Should this one too?

Reply:
Lots of more changed but fixed in this: bfa9b9f